.env.example

```diff
@@ -10,6 +10,7 @@ SUPABASE_KEY=
 AUTO_PAY_MODE=
 ANALYTICS_MODE=
 
-# Use `trace` to get verbose logging or `info` to show less
+# Log environment
+LOG_ENVIRONMENT=production # development to see logs in console
 LOG_LEVEL=debug
```

Comment:
What are all the options perhaps you can include in a comment? I would imagine that the default level would not be `debug` (for production use)

Reply:
```suggestion
LOG_LEVEL=debug # error, warn, info, http, verbose, debug, silly
```

What is silly? 👀

Reply:
```suggestion
LOG_LEVEL=debug # 0: error 1: warn 2: info 3: http 4: verbose 5: debug 6: silly
```